src/packages/console/src/pages/createSitePage.tsx

```diff
@@ -143,99 +108,95 @@ export const CreateSitePage = (): React.ReactElement => {
+          <Stack.Item growthFactor={1} shrinkFactor={1} />
+          <Box variant='card'>
+            <Stack direction={Direction.Vertical} shouldAddGutters={true}>
+              <Stack.Item alignment={Alignment.Center} gutterAfter={PaddingSize.Wide2}>
+                <Text tag='h1' variant='header5'>Create a new site</Text>
+              </Stack.Item>
+              {accounts === null ? (
+                <Text tag='p'>
+                  {'An error occurred. Please try again later.'}
+                </Text>
+              ) : (
+                <Form isLoading={isLoading || accounts === undefined} onFormSubmitted={onCreateSiteClicked}>
+                  <Stack direction={Direction.Vertical} shouldAddGutters={true} isFullWidth={true}>
+                    <FormControl
+                      variant='outlined'
+                      margin='normal'
+                      required
+                      fullWidth
+                    >
+                      <InputLabel id='account-select-label'>Account</InputLabel>
+                      <Select
+                        labelId='account-select-label'
+                        value={selectedAccountId}
+                        onChange={onAccountSelected}
+                        error={selectedAccountIdError !== undefined}
+                        label='Account'
+                        // helperText={selectedAccountIdError}
+                      >
+                        {accounts?.map((account: Account, index: number): React.ReactElement => (
+                          <MenuItem key={index} value={account.accountId}>{account.name}</MenuItem>
+                        ))}
+                      </Select>
+                    </FormControl>
+                    <SingleLineInput
+                      id='slug'
+                      label='Site slug'
+                      name='slug'
+                      value={slug}
+                      onValueChanged={onSlugChanged}
+                      inputType={InputType.Text}
+                      inputWrapperVariant={slugError ? 'error' : ''}
+                      messageText={slugError}
+                      placeholderText='Site slug'
+                    />
+                    <Text variant='small' alignment={TextAlignment.Left}>
+                      {!slug ? 'This will be your everypage sub-domain e.g. hello.evrpg.com' : `Your everypage sub-domain will be ${slug}.evrpg.com`}
+                    </Text>
+                    <SingleLineInput
+                      id='name'
+                      label='Site name'
+                      name='name'
+                      value={name}
+                      onValueChanged={onNameChanged}
+                      inputType={InputType.Text}
+                      inputWrapperVariant={nameError ? 'error' : ''}
+                      messageText={nameError}
+                      placeholderText='Site name'
+                    />
+                    <SingleLineInput
```

Comment:
this one used to actually be a select its just wierdly done via a textfield

Reply:
Instead of a input, we can have a button

Reply:
or a custom styled component

Reply:
well if you can make a custom styled component dropdown that would be the best cos we can turn it into a ui-react component later. but button wouldnt work cos you have to pick from a list of options

Reply:
Actually I think a button or a hybrid will be good. As it opens up a model

Reply:
like a text+button
the currently selected templete is set as text
and when button is clicked it opens the model

Reply:
oh yes it opens a modal i forgot. yes try that.

Reply:
Ok

Reply:
Should I make a styled component

Reply:
or try to use the existing components in ui-react

Reply:
what should I use to create a flex div using ui-react

Reply:
depends what you want to do with it but generally stack